cmake/CmakeFunctionHelper.cmake

```diff
@@ -424,6 +429,41 @@ elseif(${TARGET} STREQUAL "ARM_GPU")
     "float"
   )
   foreach(data ${supported_types})
+  if(${MODEL} STREQUAL "RESNET_50")
+  add_gemm_configuration(
```

Comment:
Needs one level of indentation for the `if` and another for the body

Reply:
In general it seems like the CMake formatting needs a bit more consistency.

Reply:
I'll give it a pass to make it more consistent! I'd like to have automatic formatting but the only one I found split the gemm configuration parameters to a single line for each which is pretty nasty.